src/hooks/useValidateRepository.ts

```diff
@@ -15,16 +30,17 @@ const branchSearchString = 'branch:';
 const pathSearchString = 'path:';
 const endSearchString = ', context';
 
-const getErrorMessage = (rawErrorString: string, searchString: string) =>
-  rawErrorString.slice(
+const getErrorMessage = (rawErrorString: string, searchString: string) => {
+  return rawErrorString.slice(
     rawErrorString.indexOf(searchString) + searchString.length,
     rawErrorString.indexOf(endSearchString)
   );
+};
 
 const useValidateRepository = (
   getFieldValue: (name: NamePath) => string,
-  setValidationState: React.Dispatch<any>,
-  validateRepository: any
+  setValidationState: React.Dispatch<React.SetStateAction<ValidationState>>,
```

Comment:
It may be just regular function with these data:
```suggestion
  setValidationState: (state: ValidationState) => void,
```